packages/content/templates/nuxt-content.js

```diff
@@ -37,6 +39,10 @@ function propsToData (node, doc) {
       obj[field] = evalInContext(value, doc)
       data.on = data.on || {}
       data.on[event] = e => doc[value] = processor(e)
+    } else if (rxClass.test(key)) {
```

Comment:
Creating an extra condition for `class` and `style` is redundant. Since there is `rootKeys` variable in [line 3](https://github.com/nuxt/content/blob/main/packages/content/templates/nuxt-content.js#L3) that detects root attributes like `class` and `style`.

The issue comes from attribute key. Remark create `class` attribute with `className` key and according to `rootKeys` array, `className` does not detects as root attribute. Adding `className` into `rootKeys` array in [line 3](https://github.com/nuxt/content/blob/main/packages/content/templates/nuxt-content.js#L3) will solve the issue.

```
 - const rootKeys = ['class-name', 'class', 'style']
 + const rootKeys = ['class-name', 'class', 'className', 'style']
``` 

As for the `style`, I believe it works as expected like `class` and there is no need to create custom logic for it.

Reply:
@farnabaz 

Thank you for pointing it out!
Yeah, actually I thought it was redundant, but couldn't come up with adding `className`. 😇 🙏 

I ran the test again and it passed (at least `component.test.js` since `highlighter.spec.js` has been failed recently)

Reply:
Thanks 🙂